vignettes/delta-method.Rmd

```diff
@@ -119,3 +119,172 @@ Take the second derivative by subtracting each of the levels from the base level
 #### Continuous Variables
 
 Compute the second derivative of the link function and use that in place of the first derivative above. You need to explicitly compute the second derivative because you want an instantaneous rate of change as opposed to the rate of change over a range as with categorical variables above.
+
+# Example
+
+Say we're fitting a logistic regression using the `margex` data, and we're interested in the predicted outcome for different treatment groups:
+
+```{r}
+library(modmarg)
+data(margex)
+lg <- glm(outcome ~ treatment * age, data = margex, family = 'binomial')
+summary(lg)
+```
+
+where the `treatment` variable $\tau \in {0, 1}$. What is the average outcome when $\tau = 0$ or $\tau = 1$?
+
+Based on these results alone, we don't really know what the treatment effect is, both because we can't directly interpret the coefficients, because there's an interaction term, and because the effect of each covariate depends on the levels of the other covariates.
+
+## Point estimates
+
+To get an estimate of the average levels of the outcome for different values of treatment, we can set treatment to 0 or 1 for the entire dataset, generate predicted outcomes for the dataset, and averaging the results (in other words, what would the outcome have been, if everybody was in the control / treatment group?). We'll use the `predict` and `plogis` functions to do this. Predict will calculate the linear predictors (in this case, setting $\tau = t$: 
+\[
+\hat{y}_i = \alpha + \beta_1 \cdot t + \beta_2 \cdot \text{age}_i + \beta_3 \cdot (t \cdot \text{age}_i)
+\]
+`plogis` is the inverse link function $f(z) = \frac{1}{1 + \exp(-z)}$, which transforms the linear predictors to the scale of the outcome variable (predicted probabilities). Finally, we can find the average predicted level.
+
+Combining these, we have:
+\[
+g(\hat{y}_{\tau = t}) = \frac{1}{n} \sum_{i = 1}^n (1 + \exp(-\beta x_{i, \tau = t}))^{-1}
+\]
```

Comment:
Changed this to `g(yhat)` throughout since I realized I refer to `yhat` as the linear predictor above, and otherwise that's confusing, plus it should be emphasized that this is a function.